Taskfile.yaml

```diff
@@ -109,3 +110,16 @@ tasks:
     desc: "Check copyright"
     cmds:
       - go run {{ .VALIDATION_FILES }} --type copyright
+
+  cve:bin:
+    cmds:
+      - |
+        curl -LO https://github.com/aquasecurity/trivy/releases/download/v{{.TRIVY_VERSION}}/trivy_{{.TRIVY_VERSION}}_Linux-64bit.tar.gz
+        tar -zxvf trivy_{{.TRIVY_VERSION}}_Linux-64bit.tar.gz -C /opt/
+        mv /opt/trivy /usr/local/bin/trivy
+        chmod +x /usr/local/bin/trivy
+
+  cve:scan:
+    cmds:
+      - task cve:bin
+      - tools/cve/scan-main.sh {{.REPORT_FILE_NAME}} pr375
```

Comment:
Should it be 'main'?  Don't forget to change pr375 before merging.

Reply:
Yes, i remember this)